apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,29 +625,31 @@ var ConversationView = {
     // to slide correctly. Bug 1009541
     this.cancelEdit();
 
-    if (Navigation.isCurrentPanel('thread')) {
```

Comment:
__optional note:__ if I were you I'd rather to not touch it since it has little to no relation to navigation patch anyway :)